genai-perf/tests/test_exporters/test_json_exporter.py

```diff
@@ -253,6 +253,7 @@ class TestJsonExporter:
           "tokenizer_trust_remote_code": false,
           "verbose": false,
           "goodput": null,
+          "header": null,
```

Comment:
Does it make sense to make this null?
I can see this being used often, based on the same logic that we promoted it. We likely need to ensure its exporting correctly.
What do you think?

Reply:
I think this is the default value. Are you asking for it to be exported differently or to set the header in the unit test? I don't expect it to be exported this way, just tested in parallel with the way we test the other fields.

In general, we might want to consider how to unit test this exporter better. Right now, we have a few unit tests to cover all of the args and we just sort of add the args to this expected JSON result every time we add an arg to the parser.

Reply:
like many things, this quickly outgrew its initial design.
im fine leaving this alone for now but we need to revisit once we move to a config file.